src/resources/resources.repository.ts

```diff
@@ -4,5 +4,13 @@ import { EntityRepository } from "@mikro-orm/postgresql";
 
 import { Resources } from "@/common/entities/resources.entity";
 
+import { CreateResourceDto } from "./resources.dtos";
+
 @Injectable()
-export class ResourcesRepository extends EntityRepository<Resources> {}
+export class ResourcesRepository extends EntityRepository<Resources> {
+  async createOne(resourceData: CreateResourceDto): Promise<Resources> {
```

Comment:
```suggestion
  async createOne(createResourceDto: CreateResourceDto): Promise<Resources> {
```

[NIT] - Probably better to keep the names consistent wherever we can